docs/pages/blogs/file-cache.mdx

```diff
@@ -0,0 +1,202 @@
+# A better Python cache for slow function calls
+**William Zeng** - March 18th, 2024
+
+---
+We wrote a file cache - it's like Python's `lru_cache`, but it stores the values in files instead of in memory. This has saved us hours of time running our LLM benchmarks, and we'd like to share it as its own Python module. Thanks to [Luke Jaggernauth](https://github.com/lukejagg) (former Sweep engineer) for building the initial version of this!
+
+Here's the link:
+https://github.com/sweepai/sweep/blob/main/docs/public/file_cache.py. To use it, simply add the `file_cache` decorator to your function. Here's an example:
+```python
+import time
+from file_cache import file_cache
+
+@file_cache()
+def slow_function(x, y):
+    time.sleep(30)
+    return x + y
+
+print(slow_function(1, 2)) # -> 3, takes 30 seconds
+print(slow_function(1, 2)) # -> 3, takes 0 seconds
+```
+
+## Background
+We spend a lot of time on prompt engineering our agents at Sweep. Our agents take a set of input strings, format them as a prompt, then extract the important output from them. We chain multiple agents to turn a GitHub issue to a pull request.
+For example, to modify code we'll input the old code, relevant context, and instructions then output the new code.
```

Comment:
I think, we need a bit more detail for this part, i.e what is relevant context, I think someone unfamiliar with Sweep, will be a bit confused about exactly what this is.